website/docs/language/settings/backends/azurerm.html.md

```diff
@@ -247,6 +253,10 @@ When authenticating using AzureAD Authentication - the following fields are also
 
 -> **Note:** When using AzureAD for Authentication to Storage you also need to ensure the `Storage Blob Data Owner` role is assigned.
 
+* `use_microsoft_graph` - (Optional) Should Microsoft Graph (MSAL) be used instead of Azure Active Directory Graph (ADAL) for authentication? Defaults to `false`.
```

Comment:
```suggestion
* `use_microsoft_graph` - (Optional) Should MSAL be used for authentication instead of ADAL, and should Microsoft Graph be used instead of Azure Active Directory Graph? Defaults to `false`.
```